test/depositERC20.js

```diff
@@ -35,6 +35,67 @@ contract('Dead', (accounts) => {
       assert(deadFinalBalance.eq(ownerInitialBalance),
         `${accountabilityFail}. the DMS' final balance was not properly incremented`);
     });
+
+    it('should not allow a deposit larger than the sender\'s balance', async () => {
+      const dead = await Dead.deployed();
+      const token = await Token.deployed();
+      const errMsg = 'the owner was able to deposit with an amount greater than their initial balance';
+
+      //Record the initial balances of both accounts
+      const ownerInitialBalance = await token.balanceOf.call(owner);
+      const dmsInitialBalance = await token.balanceOf.call(dead.address);
+
+      //Attempt to deposit more tokens to the DMS than allowed
+      const attemptDepositAmount = ownerInitialBalance + 1;
```

Comment:
`ownerInitialBalance` is going to be a bignumber. Because javascript is terrible, I don't know exactly what type of behavior using the plus operator is going to produce... So to be safe, do `ownerInitialBalance.add(new BN('1', 10))`